sdk/identity/src/lib.rs

```diff
@@ -51,4 +51,4 @@ mod errors;
 pub use errors::Error;
 pub mod refresh_token;
 pub mod token_credentials;
-mod traits;
+pub mod traits;
```

Comment:
I don't think we want to expose the `traits` mod, but rather re-export from the top level?

```suggestion
pub use traits::{BearerToken, RefreshToken};
```

Reply:
Sounds good.  Note, I added the ExtExpiresIn trait as well to your suggested list.